wandb/sdk/data_types/base_types/media.py

```diff
@@ -27,6 +27,14 @@
 def _wb_filename(
     key: Union[str, int], step: Union[str, int], id: Union[str, int], extension: str
 ) -> str:
+    # Avoid writing to absolute paths by striping any leading slashes.
+    key = str(key).lstrip(os.sep)
+
+    # Avoid directory traversal by replacing dots with underscores.
+    keys = key.split(os.sep)
+    keys = [k.replace(".", "_") if k in (os.curdir, os.pardir) else k for k in keys]
+    key = os.sep.join(keys)
```

Comment:
for now it's ok to keep it like this, but probably would want to document this behavior somewhere